packages/arb-avm-cpp/avm_values/include/avm_values/opcodes.hpp

```diff
@@ -487,12 +487,12 @@ const std::unordered_map<OpCode, uint64_t> InstructionArbGasCost = {
     {OpCode::DEBUG_PRINT, 1},
 
     {OpCode::NEW_BUFFER, 1},
-    {OpCode::GET_BUFFER8, 10},
-    {OpCode::GET_BUFFER64, 10},
-    {OpCode::GET_BUFFER256, 10},
-    {OpCode::SET_BUFFER8, 100},
-    {OpCode::SET_BUFFER64, 100},
-    {OpCode::SET_BUFFER256, 100},
+    {OpCode::GET_BUFFER8, 3},
+    {OpCode::GET_BUFFER64, 3},
+    {OpCode::GET_BUFFER256, 3},
+    {OpCode::SET_BUFFER8, 240},
```

Comment:
Best to leave a comment saying these are placeholders, and the real gas cost will be computed dynamically based on the state of the buffer.